services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/repository/UserGroupsRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.sourcefuse.jarc.services.usertenantservice.repository;
+
+import com.sourcefuse.jarc.services.usertenantservice.DTO.UserGroup;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface UserGroupsRepository extends JpaRepository<UserGroup, UUID> {
+
+    void deleteAllByGroupId(UUID groupId);
+
+    @Query(value = "select * from main.user_groups where group_id=:groupId and user_tenant_id=:userTenantId LIMIT 1" ,nativeQuery = true)
+    List<UserGroup> findByGpIDUsrTntId(UUID groupId, UUID userTenantId);
```

Comment:
function name is not readable, it can be bettter

Reply:
done